pkg/util/testUtil.go

```diff
@@ -23,25 +23,18 @@ import (
 	"io/ioutil"
 	"net"
 	"net/http"
-	"strings"
 	"testing"
 	"text/template"
 	"time"
 
-	"gopkg.in/yaml.v2"
-
 	"github.com/maistra/maistra-test-tool/pkg/util/log"
-	"github.com/maistra/maistra-test-tool/pkg/util/test"
+	template2 "github.com/maistra/maistra-test-tool/pkg/util/template"
```

Comment:
Why imported with alias?

Reply:
We import `text/template` on line 27. But don't worry about the alias. It'll go away when we finish refactoring all the tests and remove the `RunTemplate()` function.